config/default/migrate_plus.migration.hs_localist_individual.yml

```diff
@@ -284,6 +288,23 @@ process:
       delimiter: .
     -
       plugin: array_pop
+  urlname_trimmed:
+    -
+      plugin: substr
+      source: urlname
+      start: 0
+      length: 220
```

Comment:
some really long names?

Reply:
Some were getting pretty long yea lol. The file extension + photo id is about 20 characters.